control-plane/api-gateway/gatekeeper/gatekeeper_test.go

```diff
@@ -35,13 +35,26 @@ var (
 	dataplaneImage      = "hashicorp/consul-dataplane"
 	name                = "test"
 	namespace           = "default"
-	labels              = map[string]string{
+
+	labels = map[string]string{
 		"component":                              "api-gateway",
 		"gateway.consul.hashicorp.com/name":      name,
 		"gateway.consul.hashicorp.com/namespace": namespace,
 		createdAtLabelKey:                        createdAtLabelValue,
 		"gateway.consul.hashicorp.com/managed":   "true",
 	}
+
+	// TODO: This could probably be handled better
```

Comment:
Open to suggestions, I did not want to completely restructure how these tests are done but do not love the repeating of values here.